contrib/npm/README.md

```diff
@@ -0,0 +1,77 @@
+# Hurl
+
+Hurl is a command line tool that runs <b>HTTP requests</b> defined in a simple <b>plain text format</b>.
```

Comment:
I would add "written in Rust" to this sentence, so there's no confusion.  I would also follow-up with a second sentence that says something like:

> The @orangeopensource/hurl package allows JavaScript developers to use Hurl in `npm` scripts.